corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -100,12 +100,15 @@ $(document).on("ajaxStart", function () {
     tfLoadingComplete();
 });
 
-FormplayerFrontend.on('showError', function (errorMessage, isHTML) {
+FormplayerFrontend.on('showError', function (errorMessage, isHTML, removeStep) {
     if (isHTML) {
         showHTMLError(errorMessage, $("#cloudcare-notifications"));
     } else {
         showError(errorMessage, $("#cloudcare-notifications"));
     }
+    if (removeStep) {
```

Comment:
it feels weird to have the `showError` trigger a back movement. how do you feel about just having the normal `showError` and then afterward call `navigation:back`